doc/source/security/using-kmip.rst

```diff
@@ -0,0 +1,98 @@
+.. _using-kmip:
+
+==========================================================
+Using the Key Management Interoperability Protocol (KMIP)
+==========================================================
+
+This feature is **technical preview** quality.
+
+**Percona Server for MySQL** 8.0.27-18 adds support for Key Management Interoperability Protocol (KMIP).
+
+KMIP enables communication between key management systems and the database server. The protocol can do the following:
+
+* Streamline encryption key management
+
+* Eliminate redundant key management processes
+
+
+
+Component installation
+--------------------------------------
+
+Only one keyring plugin or one keyring component should be enabled at a time. Enabling multiple keyring plugins or keyring components are not supported and may result in data loss.
+
+.. note::
+
+    At one time, only one keyring component or plugin should be enabled. Multiple components or plugins being enabled at the same time is not supported and may have unexpected results.
+
+Install Component
+---------------------------
+
+The KMIP component must be installed. A keyring component is not loaded with the ``--early-plugin-load`` option on the server. The server uses a manifest and the component consults its configuration file during initialization. 
+
+To install a keyring component, you must do the following:
+
+1. Write a manifest in a valid JSON format
+2. Write a configuration file
+
+You should create a global manifest file named ``mysqld.my`` in the installation directory and, optionally, create a local manifest file, also named ``mysqld.my`` in the data directory.
+
+This is an example of a local configuration file:
```

Comment:
This could also be an example of the global configuration file (if not using a local config), both allow the same values.

The only difference is that the global configuration also allows the "read_local_config" parameter, and if it is true, it will read the local configuration. (in this case, it will ignore anything set in the global config)

Reply:
Updated.